### PR TITLE
fix: Revocation error handling in rp

### PR DESCRIPTION
**Pull request by talwinder50**
Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: scripts/sandbox_start.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ healthCheck()
 # for now we wait until nginx pick up the VIRTUAL_HOST
 sleep 3
 n=0
-maxAttempts=180
+maxAttempts=200
```
Comment (fqutishat) [inline scripts/sandbox_start.sh:76]: Why this change

Comment (talwinder50) [reply scripts/sandbox_start.sh:76]: for my local this time always work .. especially for demo resolver .. otherwise it fails once n work second time.. @aholovko also mentioned similar problem.

**Review by fqutishat — APPROVED** [review]
